Header header
string parent_frame_id
sensor_msgs/Image image
graspberry_perception/ObjectAnnotation[] annotations
string[] class_labels
